pkg/gui/presentation/commits.go

```diff
@@ -73,10 +74,18 @@ func getFullDescriptionDisplayStringsForCommit(c *models.Commit, cherryPickedCom
 
 	truncatedAuthor := utils.TruncateWithEllipsis(c.Author, 17)
 
-	return []string{shaColor.Sprint(c.ShortSha()), secondColumnString, yellow.Sprint(truncatedAuthor), tagString + defaultColor.Sprint(c.Name)}
+	commitName := ""
+	if parseEmoji {
+		commitName = emoji.Sprintf(c.Name)
+	} else {
+		commitName = c.Name
+	}
+	// return []string{shaColor.Sprint(c.ShortSha()), secondColumnString, yellow.Sprint(truncatedAuthor), tagString + defaultColor.Sprint(c.Name)}
+
+	return []string{shaColor.Sprint(c.ShortSha()), secondColumnString, yellow.Sprint(truncatedAuthor), tagString + commitName}
```

Comment:
Why did you remove `defaultColor.Sprint(..)`?

Reply:
@mjarkk thank you for the review!
sorry forgot to remove comment out 🙇 
fixed https://github.com/jesseduffield/lazygit/pull/1372/commits/eac9b2f6baee060b445633d2badf8fd8fa877710